core/variables.js

```diff
@@ -488,7 +488,7 @@ Blockly.Variables.renameVariable = function(workspace, variable,
   var validate = Blockly.Variables.nameValidator_.bind(null, varType);
 
   var promptText = promptMsg.replace('%1', variable.name);
-  Blockly.prompt(promptText, '',
+  Blockly.prompt(promptText, variable.name,
```

Comment:
I think that for cloud variables, the default value in the prompt should be the variable name without the cloud prefix; this is slightly different from the scratch 2.0 behavior, but I think would ultimately lead to less confusion. Proposing code changes below:

Reply:
```suggestion
  var promptDefaultText = variable.name;
  if (variable.isCloud && variable.name.indexOf(Blockly.Variables.CLOUD_PREFIX) == 0) {
    promptDefaultText = promptDefaultText.substring(Blockly.Variables.CLOUD_PREFIX.length);
  }

  Blockly.prompt(promptText, promptDefaultText,
```

Reply:
And even though this isn't directly related to this PR, maybe [below,](https://github.com/LLK/scratch-blocks/pull/1807/files#diff-5566718ada8dbcce77f03a181262090dR495) you can make a similar optimization to existing code where we use `newName.substring(2)`, that should probably also change to `newName.substring(Blockly.Variables.CLOUD_PREFIX.length)` so that the `2` isn't hardcoded.